presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -401,7 +402,17 @@ protected Void visitAllColumns(AllColumns node, Integer context)
         protected Void visitTable(Table node, Integer indent)
         {
             builder.append(formatName(node.getName()));
+            if (node.getTableVersionExpression().isPresent()) {
+                builder.append(" " + node.getTableVersionExpression().get());
+            }
+
+            return null;
+        }
 
+        @Override
+        protected Void visitTableVersion(TableVersionExpression node, Integer indent)
+        {
+            builder.append("FOR " + node.getTableVersionType().name() + " AS OF " + formatExpression(node.getAsOfExpression(), parameters));
```

Comment:
Is function `visitTableVersion()` unnecessary as we have handled `node.getTableVersionExpression()` directly in `visitTable()`? Or should we use `process(node.getTableVersionExpression().get(), indent)` in `visitTable()`?

Reply:
@hantangwangd thanks, I have moved `visitTableVersion()` to ExpressionFormatter and changed `visitTable()`. I have verified text generation in ExpressionFormatter using `create view` case.